src/index.html

```diff
@@ -6,13 +6,61 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="/src/styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="/src/images/imac.jpeg"
```

Comment:
The `src` attribute for the image should be verified to ensure the path is correct relative to the HTML file. If the image is not displaying, check the path and adjust it accordingly.